paraview/install_pv.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Build ParaView
+if [ ! -d paraview-source ]; then
+  git clone https://gitlab.kitware.com/paraview/paraview-superbuild.git paraview-source
+  mkdir paraview-build
+  cd paraview-source
+  git checkout e9736b30ce3e50217e113080d1eeedf504c764b1
+  git submodule update --init
+  cd ..
+fi
+
+cd paraview-build
+cmake -DSUPERBUILD_PROJECT_PARALLELISM=8 \
+      -DPARAVIEW_BUILD_EDITION=CATALYST_RENDERING \
+      -Dparaview_SOURCE_SELECTION=5.8.1-RC1 \
+      -DENABLE_python=ON \
+      -DENABLE_python3=ON \
```

Comment:
We use the ParaView `superbuild here`. It builds ParaView with its dependencies (most notably python3, osmesa, ospray, nlohmann_json)

Reply:
Can we use the version of python I installed as a module? It's version 3.8.3

Reply:
Yes, you should be able to use your installed python version. Add `-DUSE_SYSTEM_python3=ON`

Reply:
Yes, you should be able to use your installed python version. Add `-DUSE_SYSTEM_python3=ON`

Reply:
Fail :(

```
[ 73%] Generating CXX resource object /home/d170/d170/nashvest/app-install/paraview/build/paraview-build/superbuild/openimagedenoise/build/weights/rt_hdr_alb_nrm.cpp
  File "/home/d170/d170/nashvest/app-install/paraview/build/paraview-build/superbuild/openimagedenoise/src/scripts/resource_to_cpp.py", line 33
    with open(in_path, 'rb') as in_file, open(out_path, 'w') as out_file:
                                       ^
SyntaxError: invalid syntax
```
A quick dig shows that the super build isn't passing the correct value of `PYTHON_INTERP` to the configure step of openimagedenoise. Any ideas?

Reply:
I remember now that this was the exact reason why I have not used your python version... Sorry, no idea!